README.md

```diff
@@ -32,7 +32,7 @@ and installing Julia, below.
 - **Binaries:** <https://julialang.org/downloads/>
 - **Source code:** <https://github.com/JuliaLang/julia>
 - **Documentation:** <https://docs.julialang.org>
-- **Packages:** <https://pkg.julialang.org/>
+- **Packages:** <https://juliahub.com> or <https://juliaobserver.com>
```

Comment:
```suggestion
- **Packages:** <https://julialang.org/packages/>
```
Which in turn links to those places—this way we won't need to update again if that list changes.

Reply:
Sure.